utils/run_model.py

```diff
@@ -165,11 +165,14 @@ def translate_over_websocket(port: int):
 
 
 def main() -> None:
-    if not os.environ.get("IS_DOCKER"):
+    dockerfile = os.environ.get("DOCKERFILE")
+    if not dockerfile:
         # Re-run the command in docker if it wasn't started.
         args = sys.argv[1:]
-        subprocess.check_call(["task", "docker-run", "--", "task", "run-model", "--", *args])
+        subprocess.check_call(["task", "docker-run-amd64", "--", "task", "run-model", "--", *args])
         return
+    if dockerfile != "local-train":
```

Comment:
AFAICT this is unreachable because if `dockerfile == "local-train"` we're in the block above that returns, or `dockerfile is None` (in which case, does the rest of this function work correctly?).

Was this meant to go above `if not Dockerfile`?